src/components/shared/C3Chart/C3Chart.js

```diff
@@ -0,0 +1,21 @@
+import c3 from 'c3';
+import React, { useState, useEffect } from 'react';
+
+const C3Chart = ({ config, element }) => {
+  const [chart, setChart] = useState();
+
+  const generateChart = (config, element) => {
+    const build = Object.assign({ bindto: '#' + element }, config);
+    setChart(c3.generate(build));
+  };
+
+  useEffect(() => {
+    if (!chart) {
+      generateChart(config, element);
+    }
+  }, [config, element, chart]);
```

Comment:
It is weird, because `generateChart()` will be executed once, no matter if these dependencies (`config`, `element`, `chart`) change or not.

Reply:
Maybe it is possible binding the chart only once time depending on `element`? This refactoring would be also useful for the cleanup.

Reply:
i changed now because if the chart exist the could change

Reply:
Si, pero esto genera un `chart.load` en cada render.

Reply:
por ahora renderiza una sola vez, tendria que cambiar el config